1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -91,27 +91,28 @@ function generate_kubernetes_config() {
       <name>openshift</name>
       <templates>
         <org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
-          <name>agent</name>
-          <image>jenkinsci/jnlp-slave</image>
+          <name>maven</name>
```

Comment:
right, we're going to be providing two slave images on dockerhub.  one that includes maven, one that includes nodejs.  This config section references them.